manifests/k/Kiwix/KiwixJS/Electron/1.9.6-E/Kiwix.KiwixJS.Electron.locale.en-GB.yaml

```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kiwix.KiwixJS.Electron
+PackageVersion: 1.9.6-E
+PackageLocale: en-GB
+Publisher: Kiwix
+PublisherUrl: https://github.com/kiwix/kiwix-js-windows
+PublisherSupportUrl: https://github.com/kiwix/kiwix-js-windows/issues
+Author: Kiwix
+PackageName: Kiwix JS PWA

```

Comment:
```suggestion
PackageName: Kiwix JS Electron
```

Reply:
@OfficialEsco It's always been "Kiwix JS PWA" because the app is an underlying PWA packaged as an Electron app. I agree that "Kiwix JS Electron" would make more sense for the Electron version. I hope it won't have any unintended effects for those upgrading.

Reply:
Wops i meant the `PackageName` seems to have changed to `Kiwix JS Electron` from `Kiwix JS PWA` in the `A`dd or `R`emove `P`rograms table, the Publisher and DisplayName field is used for matching, so if they recently just changed it it will find both PWA and Electron under as the same Package instead of loosing connection to those who have a newer version called Electron instead of PWA